src/index.html

```diff
@@ -8,7 +8,6 @@
 </head>
 <body>
   <div id="root"></div>
-
```

Comment:
why is this  space needed?

Reply:
There was an empty href label in the index that's why a deleted

Reply:
Please don't send changes in index.html

Reply:
Fixed. Add the white space that should be in the index.html